eclipse/tests/org.switchyard.tools.ui.tests/src/org/switchyard/tools/ui/tests/AbstractSwitchYardTest.java

```diff
@@ -78,4 +81,18 @@ public boolean matches(Job job) {
                     || job.getClass().getName().endsWith("JREUpdateJob"); //$NON-NLS-1$
         }
     }
+
+    protected void waitForJobs() throws OperationCanceledException, InterruptedException {
```

Comment:
FAR (For another review): we really need to put this method in a good place to factorize it